.devcontainer/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.163.1/containers/go/.devcontainer/base.Dockerfile
```

Comment:
Ignore this file, will remove - it relates to VSCode dev containers which I use locally.